python/lsst/ap/association/packageAlerts.py

```diff
@@ -36,10 +39,21 @@
 from lsst.pex.exceptions import InvalidParameterError
 import lsst.pipe.base as pipeBase
 from lsst.utils.timer import timeMethod
-
+import fastavro
+from lsst.alert.packet import Schema
+try:
+    from confluent_kafka import KafkaException
+    import confluent_kafka
+except ImportError as error:
+    error.msg += ("Could not import confluent_kafka. Alerts will not be sent to the alert stream")
 
 """Methods for packaging Apdb and Pipelines data into Avro alerts.
 """
+_ConfluentWireFormatHeader = struct.Struct(">bi")
```

Comment:
Does this really need to be a global?  Can it live in PackageAlertsTask?

Reply:
This now lives in the PackageAlertTask `_serialize_confluent_wire_header` and `_deserialize` since it isn't needed elsewhere.